packages/zostso/__tests__/__unit__/StartTsoApp.unit.test.ts

```diff
@@ -0,0 +1,114 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Session } from "@zowe/imperative";
+import { ZosmfRestClient } from "@zowe/core-for-zowe-sdk";
+import { AddressSpaceApps, StartTso, IStartStopResponses } from "../../src";
+
+const PRETEND_SESSION = new Session({
+    user: "usr",
+    password: "pasword",
+    hostname: "host.com",
+    port: 443,
+    type: "basic",
+    rejectUnauthorized: false
+});
+const MOCK_RESPONSE = Promise.resolve({
+    version: "0100",
+    reused: false,
+    timeout: false,
+    servletKey: "CUST009-123-aaaaaa",
+    queueID: "983068",
+    tsoData: [
+        { "TSO MESSAGE": { VERSION: "0100", DATA: "HELLOW exec processing has started." } },
+        { "TSO MESSAGE": { VERSION: "0100", DATA: "UNIX message queue id = 983068" } },
+        { "TSO MESSAGE": { VERSION: "0100", DATA: "Input message type = 32772" } },
+        { "TSO MESSAGE": { VERSION: "0100", DATA: "Output message type = 4" } },
+        { "TSO MESSAGE": { VERSION: "0100", DATA: "Reading application input from the UNIX message queue." } }
+    ]
+});
+
+const MOCK_START_RESPONSE: Promise<IStartStopResponses> = Promise.resolve({
+    collectedResponses: [],
+    messages: "IKJ56455I CUST009 LOGON IN PROGRESS AT 11:18:56 ON OCTOBER 14, 2024\nIKJ56951I NO BROADCAST MESSAGES\nREADY \n",
+    servletKey: "CUST009-123-aaaaaa",
```

Comment:
Same as one of Andrew's comments...
At least for unit tests.. we should use a more generic user like ZOWEUSER 😋